src/transaction.ts

```diff
@@ -651,9 +651,8 @@ export class Snapshot extends EventEmitter {
    * **Performance Considerations:**
    *
    * This method wraps the streaming method,
-   * {@link Snapshot#run} for your convenience. All rows will
-   * be stored in memory before being released to your callback. If you intend
-   * on receiving a lot of results from your query, consider using the streaming
+   * {@link Snapshot#run} for your convenience. All rows are stored in memory before being released to your callback. 
```

Comment:
Definitely running over that 80 char limit here. Does `npm run fix` correct this?